modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -299,6 +302,34 @@ Example output plugin configuration:
     Syslog_Maxsize 2048
 ```
 
+==== Datadog Configuration Parameters
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Datadog Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | The host name of the remote Datadog server. | Yes | `http-intake.logs.datadoghq.com`
+| Api Key | The Datadog API key. | Yes |
+| Compress | Compresses the payload in GZIP format. Datadog recommends setting this to `gzip`. | No | `gzip`
+| Datadog Service | The name of the service generating the logs. For example, `runtime_fabric` | No |
+| Datadog Source | The type of service. For example, `postgres` or `nginx` | No |
+| Datadog Tags | The tags assigned to the logs in Datadog. | No |
+| TLS | Enables or disables TLS support. | No | Off
```

Comment:
nit: `Enables or disables TLS support.` -> `End-to-end security communications security protocol support.`

Reply:
Ya that makes more sense, specially for datadog case